pjax_rails.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir['lib/**/*.rb', 'lib/**/*.js', 'vendor/**/*.js']
 
-  s.add_dependency 'railties', '>= 3.2', '< 6.0'
+  s.add_dependency 'railties', '>= 3.2'
```

Comment:
I don't think this is necessary, especially when rails 6.0 isn't released

Reply:
The gemspec for all the rails components (activerecord, activemodel, etc) are listed as version 6.0.0.alpha which prevents this from being used if left with an upper limit.